### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.4 - es. 1 (14 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#289] [Esercizi M-Z]
![es 1](https://github.com/user-attachments/assets/919fde03-b5d2-4ba5-a564-14e919c42fa4)

**Comment by marcoseverini** [comment]
<img width="917" alt="immagine" src="https://github.com/user-attachments/assets/9e61b99c-a42e-44c1-ae5d-662c6dc0735e">
<img width="812" alt="immagine" src="https://github.com/user-attachments/assets/40137a3d-c1e3-4323-9a45-8e886e47e4d9">
<img width="902" alt="immagine" src="https://github.com/user-attachments/assets/75d535cc-0d4a-47d0-ac11-170aed180d12">
